chrome/content/zotero/fileInterface.js

```diff
@@ -510,14 +510,17 @@ var Zotero_File_Interface = new function() {
 			format = "rtf";
 		}
 		
+		// determine locale preference
+		var locale = io.locale;
+		
 		// generate bibliography
 		try {
 			if(io.method == 'copy-to-clipboard') {
-				copyItemsToClipboard(items, io.style, false, io.mode === "citations");
+				copyItemsToClipboard(items, io.style, locale, false, io.mode === "citations");
```

Comment:
Fixed in https://github.com/rmzelle/zotero/commit/6983a99f0b59746ae1c28e6823f0b642610ec9cc